tests/manage/z_cluster/pvc_ops.py

```diff
@@ -0,0 +1,241 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.testlib import ignore_leftovers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+def pods_creation(pod_factory, pvc_objs):
+    """
+    Create pods
+
+    Args:
+        pvc_objs (list): List of ocs_ci.ocs.resources.pvc.PVC instances
+        pvc_objs (function): Function to be used for creating pods
+
+    Returns:
+        list: list of Pod objects
+    """
+    pod_objs = []
+
+    # Create one pod using each RWO PVC and two pods using each RWX PVC
+    for pvc_obj in pvc_objs:
+        if pvc_obj.volume_mode == 'Block':
+            pod_dict = constants.CSI_RBD_RAW_BLOCK_POD_YAML
+            raw_block_pv = True
+        else:
+            raw_block_pv = False
+            pod_dict = ''
+        if pvc_obj.access_mode == constants.ACCESS_MODE_RWX:
+            pod_obj = pod_factory(
+                interface=pvc_obj.interface, pvc=pvc_obj, status="",
+                pod_dict_path=pod_dict, raw_block_pv=raw_block_pv
+            )
+            pod_objs.append(pod_obj)
+        pod_obj = pod_factory(
+            interface=pvc_obj.interface, pvc=pvc_obj, status="",
+            pod_dict_path=pod_dict, raw_block_pv=raw_block_pv
+        )
+        pod_objs.append(pod_obj)
+
+    return pod_objs
+
+
+def create_pvcs(
```

Comment:
same, this function must exist somewhere in the project, please use existing one

Reply:
I checked,  did not get existing function to used multi_pvc_factory().